src/styles/unstyled.anchor.ts

```diff
@@ -0,0 +1,16 @@
+import { styled } from '@linaria/react'
+
+export const unstyledAnchor = styled.a`
```

Comment:
Firstly would need to be `ElUnstyledAnchor` as it creates a class and all our classes are `el-` prefix.

Secondly, we don't want them to inherit as that comes from the parent. You are using this in combination with another class so might override, might not. Depends on the specificity and when the CSS is loaded. 

Think in this case just easier to copy-paste the styles in TBH, being carful not to override anything already there. Only really being applied to a few components - not worth making re-usable. 

I'd just add to breadcrumb, accordion etc the things you need to stop the anchor froma adding styles and be done with it!